features/buyer/tell_us_about_contract.feature

```diff
@@ -119,12 +119,11 @@ Scenario: User does not award contract as there are no suitable services
   Then I see a success banner message containing 'You’ve updated ‘my cloud project’'
   And I see 'No suitable services found' text on the page
 
-Scenario: User is still assessing services
+Scenario: User is still assessing services - via the saved searches dashboard
   Given I am logged in as a buyer user
   And I have created and ended a search called 'my cloud project'
   And I have downloaded the search results as a file of type 'ods'
-  And I click the 'Return to overview' link
-  And I am on the 'my cloud project' page
+  When I visit the /buyers/direct-award/g-cloud page
   When I click the 'Tell us the outcome' link
```

Comment:
Is there going to be a `Tell us the outcome` link for each project on the G Cloud dashboard? On Preview I can only see it on the project page?

Reply:
Yes, there will be a link for each project that has ended.

Oh...sorry I forgot to mention that this change is dependent on https://github.com/alphagov/digitalmarketplace-buyer-frontend/pull/781 being merged first.

Reply:
Cool 👍  In that case, you might have to change the next step to `When I click a summary table 'Tell us the outcome' link for 'my cloud project'` (see https://github.com/alphagov/digitalmarketplace-functional-tests/blob/master/features/step_definitions/common_steps.rb#L314) - otherwise Capybara will probably complain if there is more than one link on the page with that wording.

Reply:
With the way the steps are run there will only ever be one instance of that link on the page.

I did try your suggestion but it fails as the xpath is different and if I were to change so that it works for the current tests and also mine I worry that the xpath would be too generic?

From: 
edit_link = page.all(:xpath, "//tr/*/span[normalize-space(text()) = '#{field_to_edit}']/../..//a[contains(normalize-space(text()), '#{link_name}')]")

To: 
edit_link = page.all(:xpath, "//tr//*[normalize-space(text()) = '#{field_to_edit}']/../../..//*[contains(normalize-space(text()), '#{link_name}')]")

Or is it ok? I am not sure.

Reply:
Hmm, I'm having trouble running this scenario (it's failing for me before it even gets to this step) as the Buyer FE branch above is out of date with master. I can take another look once it's merged?

Reply:
I think it will be to do with this alphagov/digitalmarketplace-buyer-frontend#781 still being outstanding. It's work done by Dilwoar that adds the "Tell us the outcome" link on the g-cloud page. This piece from me is dependent on that being merged first.

Reply:
@andrewchong - this has been now merged